examples/timer/index.ts

```diff
@@ -0,0 +1,11 @@
+import * as azure from '@pulumi/azure';
+
+const resourceGroup = new azure.core.ResourceGroup('example', { location: 'West US 2' });
+
+new azure.appservice.TimerSubscription('everyminute', {
+    resourceGroup,
+    schedule: '0 * * * * *',
```

Comment:
in awsx we also allowed a richer data structure for schedules: https://github.com/pulumi/pulumi-awsx/blob/master/nodejs/awsx/autoscaling/schedule.ts

Can we also do something like that here?